### DISCUSSION TITLE
Best way to put large data (>64K) in Milvus

### DISCUSSION THREAD

**Discussion by pycui** [milvus-io/milvus#35746] [Q&A and General discussion] [answered]
Hi looks like the string type only supports 64K bytes at the moment. I'm wondering if there is best practice of putting larger data into Milvus (other than splitting the data manually). Thanks!

**Comment by xiaofan-luan** [comment]
We don't recommend to do so.

Is there a specific reason to do so?

**Reply by pycui** [reply]
I'm looking for ways to store (small) documents with the embedding together. This can be more efficient to do document retrieval separately. If the limit is higher I think it will be a good trade-off for use cases like this.

**Reply by xiaofan-luan** [reply, marked as answer]
The best way recommend is to store a reference here in milvus.

We can do larger varchars -> like 100k, but I think it will definitely affect the performance
